client/my-sites/plugins/plugin-autoupdate-toggle/index.jsx

```diff
@@ -179,8 +177,19 @@ PluginAutoUpdateToggle.defaultProps = {
 	disabled: false,
 };
 
-export default connect( null, {
-	recordGoogleEvent,
-	recordTracksEvent,
-	togglePluginAutoUpdate,
-} )( localize( PluginAutoUpdateToggle ) );
+export default connect(
+	( state, { site, plugin } ) => {
+		const pluginStatus = getStatusForPlugin( state, site.ID, plugin.id );
+		const inProgress =
```

Comment:
See my comment here https://github.com/Automattic/wp-calypso/pull/47920/files#r533964859 but I guess that also relates with your note in the PR description about the notice actions.

Reply:
Yes. And I'll follow-up in another PR to create a selector. One step at a time 😉

Reply:
Addressed in #47951.